pkg/store/cache/inmemory.go

```diff
@@ -52,9 +53,9 @@ type InMemoryIndexCache struct {
 // InMemoryIndexCacheConfig holds the in-memory index cache config.
 type InMemoryIndexCacheConfig struct {
 	// MaxSize represents overall maximum number of bytes cache can contain.
-	MaxSize Bytes `yaml:"max_size"`
+	MaxSize model.Bytes `yaml:"max_size"`
```

Comment:
sorry for being late to the party, but why we are not using the same thing as Prometheus use, so ` units.Base2Bytes`?

Reply:
`model.Bytes` is based on `units.Base2Bytes` but adds YAML marshalling support. Am I missing anything?